src/library.js

```diff
@@ -4647,6 +4647,21 @@ LibraryManager.library = {
     else return lengthBytesUTF8(str);
   },
 
+  emscripten_wasm_return_address__deps: ['emscripten_get_callstack_js'],
```

Comment:
please add a comment explaining what this is. in particular from the name it's not obvious what it returns exactly.